manager/controllers/app/moduleinstance.go

```diff
@@ -183,6 +183,7 @@ func (r *M4DApplicationReconciler) SelectModuleInstancesPerDataset(item modules.
 	readInstructions := make([]app.ReadModuleArgs, 0)
 	readInstructions = append(readInstructions, app.ReadModuleArgs{
 		Source:          readSource,
+		AssetName:       item.AssetID,
```

Comment:
You created AssetID for internal stuff like logs (I still differ in opinion but nevermind). This is external. Can we use the original instead?

Reply:
Changed for all cases to be copied as-is from the original resource. Does not affect validity of kubernetes resource names or the path to secret-provider.